src/server/game/Maps/Map.cpp

```diff
@@ -2950,13 +2951,23 @@ void Map::SendInitSelf(Player* player)
     // build data for self presence in world at own client (one time for map)
     player->BuildCreateUpdateBlockForPlayer(&data, player);
 
+    // build and send self update packet before sending to player his own auras
+    data.BuildPacket(&packet);
+    player->SendDirectMessage(&packet);
+
+    // send to player his own auras (this is needed here for timely initialization of some fields on client)
+    player->SendAurasForTarget(player);
+
+    // clean buffers for further work
+    packet.clear();
+    data.Clear();
```

Comment:
is this a codestyle typo?

Reply:
> is this a codestyle typo?

what you mean?

Reply:
+    packet.clear(); < not capital letter
+    data.Clear(); < capital letter

Reply:
> * packet.clear(); < not capital letter
> * data.Clear(); < capital letter

No, it's just that methods in different classes are called differently.
`ByteBuffer::clear()` and `UpdateData::Clear()`

Reply:
never mind then. it just looks untidy to me :P